firefox_ui_tests/remote/security/test_ev_certificate.py

```diff
@@ -0,0 +1,118 @@
+        self.assertEqual(self.identity_popup.popup.get_attribute('className'),
+                         'verifiedIdentity')
+
+        # Check that the lock icon is visible
+        self.assertNotEqual(self.identity_popup.encryption_icon.
+                            value_of_css_property('list-style-image'), 'none')
+
+        # Check the certificate's domain name
+        # Bug 443116 - Larry strips the 'www.' from the host name using the eTLDservice.
+        # This is expedted behavior.
+        self.assertEqual(self.identity_popup.host.get_attribute('textContent'),
+                         self.security.get_domain_from_common_name(cert['commonName']))
+
+        # Check the organization name
+        self.assertEqual(self.identity_popup.owner.get_attribute('textContent'),
+                         cert['organization'])
+
+        # Check the owner location string
+        # More information:
+        # hg.mozilla.org/mozilla-central/file/eab4a81e4457/browser/base/content/browser.js#l7012
+        location = self.browser.get_property('identity.identified.state_and_country')
+        location = location.replace('%S', address['state'], 1).replace('%S', address['country'])
+        location = address['city'] + '\n' + location
+        self.assertEqual(self.identity_popup.owner_location.get_attribute('textContent'),
+                         location)
+
+        # Check the verifier
+        l10n_verifier = self.browser.get_property('identity.identified.verifier')
+        l10n_verifier = l10n_verifier.replace('%S', cert['issuerOrganization'])
+        self.assertEqual(self.identity_popup.verifier.get_attribute('textContent'),
+                         l10n_verifier)
+
+        # Check the encryption label
+        self.assertEqual(self.identity_popup.encryption_label.get_attribute('textContent'),
+                         self.browser.get_property('identity.encrypted2'))
+
+        # Open the Page Info window by clicking the More Information button
+        page_info = self.browser.open_page_info_window(
+            lambda _: self.identity_popup.more_info_button.click())
+
+        try:
+            # Verify that the current panel is the security panel
+            self.assertEqual(page_info.deck.selected_panel, page_info.deck.security)
+
+            # Verify the domain listed on the security panel
+            self.assertIn(cert['commonName'],
+                          page_info.deck.security.domain.get_attribute('value'))
+
+            # Verify the owner listed on the security panel
+            self.assertEqual(page_info.deck.security.owner.get_attribute('value'),
+                             cert['organization'])
+
+            # Verify the verifier listed on the security panel
+            self.assertEqual(page_info.deck.security.verifier.get_attribute('value'),
+                             cert['issuerOrganization'])
+
+            page_info.close()
+        finally:
+            self.browser.focus()
```

Comment:
This will not close the page info window.  You surely want to move it into finally too, just before this line.